### DISCUSSION TITLE
Data Export and Import

### DISCUSSION THREAD

**Discussion by darmau** [umami-software/umami#2159] [General]
### Data Export and Import 

At present, my umami is running on a server through docker. I want to migrate it to vercel + supabase, but I have not found the function of data import and export. If there is an import and export function, it will be very helpful for us to migrate umami.

**Comment by franciscao633** [comment]
I think for self-hosted it's best to just leverage the existing backup and restore tools available (ex: pg_dump for PostgreSQL). You can run create the dump file through your docker container and restore on any new host you desire. From there you can just update the `DATABASE_URL` and you will be set. This does have limitations, since you can't combine data between two instances, and its not an easy option for non-technical folks.

**Comment by navjotjsingh** [comment]
Is there any plan to bring this feature to the self-hosted version?

**Comment by barnabynagy** [comment]
I'm trying to migrate from pikapods to railway app. Is there a way to migrate the data like export-import?
